plugin/al/mayautils/AL/maya/utils/PluginTranslatorOptions.h

```diff
@@ -411,9 +437,17 @@ class PluginTranslatorOptions
         /// \brief  ctor
         /// \param  name the name of the option
         /// \param  defVal the default value
-        Option(const char* const name, const float& defVal)
+        Option(
+            const char* const name,
+            const float&      defVal,
+            int               defPre,
+            const char*       defController,
+            bool              defState)
```

Comment:
Same here, I'd rename defState to enableState.

Reply:
Updated.